api/app/src/external/fhir/api.ts

```diff
@@ -64,6 +64,46 @@ export class FHIRClient extends MedplumClient {
       }
     }
   }
+
+  // HAPI FHIR specific
+  private readonly HAPI_URI = "fhir/DEFAULT";
+
+  // HAPI FHIR specific
+  async createTenant(org: { organizationNumber: number; cxId: string }): Promise<void> {
+    const url = this.getBaseUrl() + this.HAPI_URI + "/$partition-management-create-partition";
+    const payload = {
+      resourceType: "Parameters",
+      parameter: [
+        { name: "id", valueInteger: org.organizationNumber },
```

Comment:
Should we just use the full org ID here? Why are we just using the org number?

Reply:
That's because of how partitions work on HAPI - [source](https://hapifhir.io/hapi-fhir/docs/server_jpa_partitioning/partitioning_management_operations.html#creating-a-partition):

> The numeric ID for the partition. This value can be any integer, positive or negative or zero. It must not be a value that has already been used.

Using the customer ID as partition name as this should be more unique than Org's ID - although they're 1:1 today, we talked about the possibility of +1 in the future. But this is NABD, I can move to Org ID if you prefer.

Reply:
Got it, this is more of a question. Using the cxId as the name makes sense 👍